configs/dest/README.md

```diff
@@ -0,0 +1,51 @@
+# DEST
+
+[DEST: Depth Estimation with Simplified Transformer](https://arxiv.org/abs/2204.13791)
+
+## Introduction
+
+<!-- [ALGORITHM] -->
+
+<a href="https://github.com/NVIDIA/DL4AGX/tree/master/DEST">Official Repo</a>
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Transformer and its variants have shown state-of-the-art results in many vision tasks recently, ranging from image classification to dense prediction. Despite of their success, limited work has been reported on improving the model efficiency for deployment in latency-critical applications, such as autonomous driving and robotic navigation. In this paper, we aim at improving upon the existing transformers in vision, and propose a method for self-supervised monocular Depth Estimation with Simplified Transformer (DEST), which is efficient and particularly suitable for deployment on GPU-based platforms. Through strategic design choices, our model leads to significant reduction in model size, complexity, as well as inference latency, while achieving superior accuracy as compared to state-of-theart. We also show that our design generalize well to other dense prediction task such as semantic segmentation without bells and whistles.
```

Comment:
Replace with the following:

Transformer and its variants have shown state-of-the-art results in many vision tasks recently, ranging from image classification to dense prediction. Despite of their success, limited work has been reported on improving the model efficiency for deployment in latency-critical applications, such as autonomous driving and robotic navigation. In this paper, we aim at improving upon the existing transformers in vision, and propose a method for Dense Estimation with Simplified Transformer (DEST), which is efficient and particularly suitable for deployment on GPU-based platforms. Through strategic design choices, our model leads to significant reduction in model size, complexity, as well as inference latency, while achieving superior accuracy as compared to state-of-the-art in the task of self-supervised monocular depth estimation. We also show that our design generalize well to other dense prediction task such as semantic segmentation without bells and whistles.

Reply:
addressed